### ISSUE TITLE
It Breaks Gnome

### ISSUE THREAD

**Issue by eliasfarah** [gonzaarcr/Fildem#52]
Hi, 

I've installed the last version, but sometimes when I click in items on the menu all the system breaks and just reboot the system to get responsive again.

This happend to me just 3 times, I'm using in production machine and must to uninstall it. 

Anyone can tell us if this bug happend to you too?

NAME="Pop!_OS"
VERSION="20.10"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.10"
VERSION_ID="20.10"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=groovy
UBUNTU_CODENAME=groovy
LOGO=distributor-logo-pop-os

Kernel 5.11

**Comment by gonzaarcr** [comment]
Wow, I’ve never had to reboot. Do you use Wayland? Do you have a file on /var/crash ?

**Comment by DanielRios549** [comment]
This is true, in my usage, what breaks is the HUD, after I try to use it, the keyboard stop responding, I even cannot restart the shell with ALT + F2, I cannot type a text in a text input, the only way to fix it is restart the computer pressing the power button or cutting of the power

**Comment by gonzaarcr** [comment]
Is that Wayland or Xorg? If you've tried rebooting, I'm guessing it's Xorg, but just to be sure.

**Comment by eliasfarah** [comment]
The same behavior @DanielRios549 happend to me. I'm working with pop os 20.10 default Sessions, Maybe xorg. I'll find out.

**Comment by eliasfarah** [comment]
> Wow, I’ve never had to reboot. Do you use Wayland? Do you have a file on /var/crash ?

**Comment by gonzaarcr** [comment]
What happens if, instead of using the hud with Alt + Space you call it typing `fildem-hud` on the terminal? Does it break?

**Comment by eliasfarah** [comment]
I will test and let you know
